docs/install-kafka-operator.md

```diff
@@ -35,74 +35,102 @@ This method uses a command-line tool of the commercial [Banzai Cloud Supertubes]
+    --namespace cert-manager \
+    --create-namespace \
+    --atomic \
+    --debug
+    ```
+
+1. Verify that cert-manager has been deployed and is in running state.
 
-    # Add the jetstack helm repo
-    helm repo add jetstack https://charts.jetstack.io
-    helm repo update
+    ```bash
+    kubectl get pods -n cert-manager
+    ```
 
-    # Install the CustomResourceDefinitions
-    kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.6.2/cert-manager.crds.yaml
+    Expected output:
 
-    # Install cert-manager into the cluster
-    # Using helm3
-    helm install cert-manager --namespace cert-manager --create-namespace --version v1.6.2 jetstack/cert-manager
+    ```bash
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    cert-manager-7747db9d88-vgggn             1/1     Running   0          29m
+    cert-manager-cainjector-87c85c6ff-q945h   1/1     Running   1          29m
+    cert-manager-webhook-64dc9fff44-2p6tx     1/1     Running   0          29m
+    ```
 
-Verify that the cert-manager pods have been created:
+### Install zookeeper-operator with Helm {#install-zookeeper-operator-with-helm}
 
-```bash
-kubectl get pods -n cert-manager
-```
+{{< kafka-operator >}} requires [Zookeeper](https://zookeeper.apache.org) for Kafka. Deploy a Zookeeper cluster if you don't already have one.
 
-Expected output:
+> Note: You are recommended to create a separate Zookeeper deployment for each Kafka cluster. If you want to share the same Zookeeper cluster across multiple Kafka cluster instances, use a unique zk path in the KafkaCluster CR to avoid conflicts (even with previous defunct KafkaCluster instances).
 
-```bash
-NAME                                      READY   STATUS    RESTARTS   AGE
-cert-manager-7747db9d88-vgggn             1/1     Running   0          29m
-cert-manager-cainjector-87c85c6ff-q945h   1/1     Running   1          29m
-cert-manager-webhook-64dc9fff44-2p6tx     1/1     Running   0          29m
-```
+1. If you are installing {{< kafka-operator >}} on Red Hat OpenShift, you need to elevate the permissions of the namespace of the Zookeeper service account.
 
-### Install Zookeeper {#install-zookeeper}
+    ```bash
+    oc adm policy add-scc-to-group anyuid system:serviceaccounts:zookeeper
+    ```
 
-Kafka requires [Zookeeper](https://zookeeper.apache.org). Deploy a Zookeeper cluster if you don't already have one.
+    In case you are using a non-default namespace for Zookeeper, you need to modify the command to take that namespace.
 
-> Note: You are recommended to create a separate Zookeeper deployment for each Kafka cluster. If you want to share the same Zookeeper cluster across multiple Kafka cluster instances, use a unique zk path in the KafkaCluster CR to avoid conflicts (even with previous defunct KafkaCluster instances).
+    ```bash
+    oc adm policy add-scc-to-group anyuid system:serviceaccount:{NAMESPACE_FOR_ZOOKEEPER_SERVICE_ACCOUNT}
```

Comment:
I believe you need the `s` on `serviceaccount`:  `oc adm policy add-scc-to-group anyuid system:serviceaccounts:zookeeper`

Reply:
probably good to add that reference to the subject reference in this doc since it's such a subtle distinction in the `add-scc-to-group` v. `add-scc-to-user` commands.  I made that mistake many times and it's completely silent in the config step.

Reply:
It looks okay to me:
```bash
-> kubectl get clusterrolebinding system:openshift:scc:nonroot-v2 -o yaml | yq

apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  creationTimestamp: "2023-03-22T12:06:02Z"
  name: system:openshift:scc:nonroot-v2
  resourceVersion: "30481047"
  uid: 5a21761b-e769-4cca-84c5-0be98715a943
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: system:openshift:scc:nonroot-v2
subjects:
  - kind: ServiceAccount
    name: prometheus-kube-prometheus-operator
    namespace: prometheus
  - kind: ServiceAccount
    name: prometheus-kube-prometheus-admission
    namespace: prometheus
```

But I will fix it for consistency reasons.

Reply:
Yep, it was a doc editing mistake when I was dealing with both, copy-pasted from the wrong one.

Fixed in https://github.com/banzaicloud/koperator-docs/compare/637c34fef67ef34bb8716243a26e30e947cf4a5a..1c81a46b46a8558d88f7a9ddf440d47796031cd0